src/test/ui/layout/valid_range_oob.rs

```diff
@@ -0,0 +1,15 @@
+// failure-status: 101
```

Comment:
ICEing isn't nice here :laughing: 

ideally we would catch this in `check_attr.rs`. Don't have to do that in this PR as we are talking about an internal attribute here.

Reply:
fwiw: I want to remove the attribute entirely.